pkg/config/logs/log.go

```diff
@@ -108,10 +109,27 @@ func SetupLogger(loggerName LoggerName, logLevel, logFile, syslogURI string, sys
 	}
 	_ = seelog.ReplaceLogger(loggerInterface)
 	log.SetupLogger(loggerInterface, seelogLogLevel)
-	scrubber.AddStrippedKeys(cfg.GetStringSlice("flare_stripped_keys"))
+	scrubber.AddStrippedKeys(MergeAdditionalKeysToScrubber(
+		cfg.GetStringSlice("flare_stripped_keys"),
+		cfg.GetStringSlice("scrubber.additional_keys")))
 	return nil
 }
 
+// MergeAdditionalKeysToScrubber merges multiple slices of keys into a single slice
+func MergeAdditionalKeysToScrubber(elems ...[]string) []string {
```

Comment:
This method should be private. Also, this should be a `slices.Clone`, an `append` and `slices.Compact`.

Reply:
ho really nice I did not know I could do it like that, I'll try thx